src/components/Button/Button.js

```diff
@@ -25,21 +23,19 @@ export type ButtonProps = {
   /** possible button types */
   type?: 'submit' | 'button' | 'reset',
   /** possible button colors */
-  color?: 'primary' | 'secondary' | 'neutral' | 'danger',
+  color?: 'primary' | 'neutral' | 'danger',
   /** the type of button */
-  variant?: 'outlined' | 'raised' | 'ghost',
```

Comment:
Why did you remove `secondary` color and `ghost` variant?

Reply:
@ej9x we never use it  if out DSM provide other button styles later - we can add it